plugins/w3c_logs.yaml

```diff
@@ -35,6 +35,47 @@ parameters:
     description: Max number of W3C files that will be open during a polling cycle
     type: int
     default: 512
+  - name: timestamp_layout
+    description: Optional timestamp layout which will parse a timestamp field
+    type: string
+  - name: timestamp_parse_from
+    description: Field to parse the timestamp from, required if 'timestamp_layout' is set
+    type: string
```

Comment:
The W3C spec is actually pretty explicit about how the timestamp should be formatted, and what fields it should be on (it's actually split between two fields, `date` and `time`). 

It's defined towards the bottom of this page: https://www.w3.org/TR/WD-logfile.html - Should we just be parsing this as defined by the spec automatically, like we do in IIS? 

I don't think these parsing options will actually work with actual W3C logs, where, again, the date and time are on separate fields that need to be joined to be properly parsed.

Reply:
While we did see, in a single log file, a non-conforming entry, I've followed your suggestion. I think I did it the right way to make this default and have it still work the other way. Pushing shortly.